### ISSUE TITLE
Example of what gas_mole_ss.out and accompanying gas_mole_tr.out

### ISSUE THREAD

**Issue by wittregr** [VlachosGroup/openmkm#21]
Can we have an example of what a gas_mole_ss.out and accompanying gas_mole_tr.out look like for a numerically solved pfr? Currently, the transient file is not outputting meaningful values. Once we have these files, we will elaborate the documentation on them.

**Comment by MaxRCohen** [comment]
Hi @mbkumar, could we get an example of this? It would be good to confirm we understand what these files entail.

**Comment by skasiraj** [comment]
@wittregr and @MaxRCohen Currently OpenMKM doesn't print transient data from PFR reactors be it PFR_0D or PFR_1D. If this is something we need, please add a new feature request. If we need to edit documentation for this please let me know.. 

This issue is a possible duplication of Issue #25 

Do you still need the example for the gas_mole_ss.out. We should be readily able to produce one from our examples.

**Comment by MaxRCohen** [comment]
Yes, we still need an example for the gas_mole_ss.out so the documentation can be updated.
